src/main/java/com/github/manolo8/darkbot/core/objects/Gui.java

```diff
@@ -16,7 +16,7 @@ public class Gui extends SpriteObject implements API, eu.darkbot.api.game.other.
     protected final Point minimized = new Point();
 
     public long addressInfo, featureWindowDefinition;
-    public boolean visible;
+    public boolean visible, isClosing;
```

Comment:
I'm against adding more public fields unless they're needed (as it makes it impossible to get rid of them later on without causing plugin exceptions), if you must, add it as private and add a getter.

We have isTweening which is for when a gui is transitioning (ie: showing the open or close animation), is this different? We have show(true) and show(false) methods which will return false until transition has finished, based on timers, does this superceed that?